### PR TITLE
Fix org.eclipse.collections.api.list api doc incorrect . Fixes #612

### LINKED ISSUE #612: org.eclipse.collections.api.list api doc incorrect
"This package contains 4 interfaces:"

It actually contains 5.

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Signed-off-by: Daniel Dias <daniel.dias.analistati@gmail.com>

this PR fix the issue #612

### CODE REVIEW

**Comment by nikhilnanivadekar** [comment]
@Daniel-Dos is there a way to auto-update all package-info files? I think these files are fairly dated and we need to update all of them. But doing them manually will be cumbersome, I was wondering if Eclipse could do a bulk auto-update?

**Comment by Daniel-Dos** [comment]
Hi @nikhilnanivadekar 

I do not know about it. I do not know if there is any Maven plugin for this.

But I agree that doing this for all files will be complicated.

**Comment by nikhilnanivadekar** [comment]
May be @leomrlima might know how to update the package-info files?

**Comment by nikhilnanivadekar** [comment]
Closing and reopening to retrigger builds

**Comment by nikhilnanivadekar** [comment]
I do not think there is an automated way to fix this. So, we will fix these on an as found basis.

File: eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/package-info.java [hunk 1]
```diff
@@ -25,6 +25,9 @@
  *     <li>
  *         {@link org.eclipse.collections.api.list.FixedSizeList} - a list that may be mutated, but cannot grow or shrink in size.
  *     </li>
+ *     <li>
+ *         {@link org.eclipse.collections.api.list.ParallelListIterable} - a {@link org.eclipse.collections.api.RichIterable} which will defer evaluation for certain methods like select, reject, collect, etc.
```
Comment (nikhilnanivadekar) [inline eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/package-info.java]: Please add this as well `Evaluation occurs in parallel`. So the final description will look like this:
`{@link org.eclipse.collections.api.list.ParallelListIterable} - a {@link org.eclipse.collections.api.RichIterable} which will defer evaluation for certain methods like select, reject, collect, etc. When evaluation occurs it is in parallel.`

**Comment by nikhilnanivadekar** [comment]
@Daniel-Dos I have suggested a small change. Once done, please squash your commits and rebase on to master. I will merge it. Thanks for your contribution!

File: eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/package-info.java (hunk 1 above)
Comment (Daniel-Dos) [reply eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/package-info.java]: Hi @nikhilnanivadekar  please review .

**Comment by Daniel-Dos** [comment]
> I do not think there is an automated way to fix this. So, we will fix these on an as found basis.

in case you have to go in package per package to check if you agree.

**Comment by nikhilnanivadekar** [comment]
I thought it would be as easy as just going to each package. But seems like the current package-info.java files contain additional data about each interface. Doing this manually will be cumbersome!

File: eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/package-info.java (hunk 1 above)
Comment (nikhilnanivadekar) [reply eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/package-info.java]: LGTM

**Comment by nikhilnanivadekar** [comment]
Thank you @Daniel-Dos for your contribution!

**Comment by Daniel-Dos** [comment]
really has a lot of information on each and some are even outdated.

Doing manual will take a lot of time.

Many thanks for the merge.

**Comment by nikhilnanivadekar** [comment]
@Daniel-Dos if you wouldn't mind raising an issue for the outdated files that you know of, we can pick them up as and when we can. Thanks!

**Pull request merged**